airflow/config_templates/default_airflow.cfg

```diff
@@ -846,6 +846,15 @@ tolerations =
 # api responses, which will cause the scheduler to hang. The timeout is specified as [connect timeout, read timeout]
 kube_client_request_args = {{"_request_timeout" : [60,60] }}
 
+
+# Optional keyword arguments to pass to the ``delete_namespaced_pod`` kubernetes client
+# ``core_v1_api`` method when using the Kubernetes Executor.
+# This should be an object and can contain any of the options listed in the ``v1DeleteOptions``
+# class defined here:
+# https://github.com/kubernetes-client/python/blob/41f11a09995efcd0142e25946adc7591431bfb2f/kubernetes/client/models/v1_delete_options.py#L19
+# Example: delete_option_kwargs = {{"grace_period_seconds": 10}}
+delete_option_kwargs =
```

Comment:
Should this default to an empty dict?

Reply:
this should be fine. Default is an empty string